Automattic/jetpack
# connection: Build JavaScript

#### Changes proposed in this Pull Request:

Instead of requiring the sources be written to ES5 syntax and loading
them directly, let's transpile them with Webpack and load them with the
Assets package like we do everything else.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check that the changes to the mirror repo make sense, i.e. new built files are included, but no unexpected files (that aren't export-ignored).
* Check that the functionality still works right, both with and without `SCRIPT_DEBUG`.
